modules/aws-installing-an-aws-load-balancer-operator.adoc

```diff
@@ -240,128 +246,112 @@ EOF
 ----
+
+. For a ROSA with HCP cluster, add the tags necessary for subnet discovery:
 +
-. Add the tags necessary for subnet discovery:
-.. Add the following `{Key: Value}` tag to the VPC hosting the
-ifndef::openshift-rosa[]
-{product-title}
-endif::openshift-rosa[]
-ifdef::openshift-rosa[]
-ROSA
-endif::openshift-rosa[]
-cluster and to all its subnets. Replace `{Cluster Infra ID}` with the Infra ID specified previously:
+.. Add the following `{Key: Value}` tag to the VPC hosting the ROSA cluster and to all its subnets. Replace `{Cluster Infra ID}` with the Infra ID specified previously:
 +
-[source, terminal]
+[source,terminal,subs="quotes,verbatim"]
 ----
-* kubernetes.io/cluster/${Cluster Infra ID}:owned
+kubernetes.io/cluster/${Cluster Infra ID}:owned
 ----
 +
 .. Add the following ELBv2 `{Key: Value}` tags to the private subnets and, optionally, to the public subnets:
-
++
 * Private subnets: `kubernetes.io/role/internal-elb:1`
 * Public subnets: `kubernetes.io/role/elb:1`
 +
 [NOTE]
 ====
-Internet-facing and internal load balancers will be created within the AZ to which these subnets belong.
+Internet-facing and internal load balancers will be created within the AWS Availability Zone to which these subnets belong.
 ====
 +
-For more information on adding tags to AWS resources, including VPCs and subnets, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tag your Amazon EC2 resources].
-+
 [IMPORTANT]
 ====
-ELBv2 resources (such as ALBs and NLBs) created by ALBO do not inherit custom tags set for
-ifndef::openshift-rosa[]
-{product-title}
-endif::openshift-rosa[]
-ifdef::openshift-rosa[]
-ROSA
-endif::openshift-rosa[]
-clusters. You must set tags separately for these resources.
+ELBv2 resources (such as ALBs and NLBs) created by AWS Load Balancer Operator do not inherit custom tags set for ROSA clusters. You must set tags separately for these resources.
 ====
+
+. Create the AWS Load Balancer Operator by completing the following steps:
 +
-. Create ALBO:
+.. Create an `OperatorGroup` object by running the following command:
 +
-[source,yaml]
+[source,terminal]
 ----
+$ cat EOF | oc apply -f -
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
   name: aws-load-balancer-operator
   namespace: aws-load-balancer-operator
 spec:
-  upgradeStrategy: Default
----
+  targetNamespaces: []
+EOF
+----
++
+.. Create a `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ cat EOF | oc apply -f -
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
   name: aws-load-balancer-operator
   namespace: aws-load-balancer-operator
 spec:
-  channel: stable-v1.0
-  installPlanApproval: Automatic
+  channel: stable-v1
   name: aws-load-balancer-operator
   source: redhat-operators
   sourceNamespace: openshift-marketplace
-  startingCSV: aws-load-balancer-operator.v1.0.0
+  config:
+    env:
+    - name: ROLEARN
+      value: "<operator_role_arn>" <1>
+EOF
```

Comment:
This is the new way of providing the role, this is good. However it means that we don't need to generate a secret for the operator anymore in the step2: `$ cat <EOF> albo-operator-aws-credentials.cfg`.

Reply:
Thanks. I removed:

> Generate the operator's AWS credentials:
> 
> [source,terminal]
> ----
> $ cat <EOF> albo-operator-aws-credentials.cfg
> [default]
> sts_regional_endpoints = regional
> role_arn = ${OPERATOR_ROLE_ARN}
> web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
> EOF
> ----